examples/Rudaina_work_area/genz_52_missing_element.py

```diff
@@ -0,0 +1,8 @@
+
+def missing_element(lst):
+
+    temp = list(range(0,(len(lst)+2)))
+    missing_elem = (sum(temp) - sum(lst))
+    return missing_elem
+
+print(missing_element([1,2,4,5,6,7]))
```

Comment:
According to PEP-8 which also defines the coding standards, we can put a space after comma ,